client/js/components/user/ToggleForm.vue

```diff
@@ -46,17 +46,14 @@ export default {
     abort () {
       if (this.formId !== '') {
         document.querySelector(`form#${this.formId}`).reset()
-      } else {
-        this.$emit('form-abort')
```

Comment:
Why were these emits deleted? Do we not need them anymore? (also see below save()).

Reply:
I could not find any usages, so I assumed they can be removed.